docs/sources/tempo/traceql/query-editor.md

```diff
@@ -11,20 +11,66 @@ keywords:
   - TraceQL
 ---
 
-# TraceQL query editor
+# Create TraceQL queries in Grafana
 
-You can use the TraceQL viewer and query editor in the Tempo data source to build queries and drill-down into result sets. The editor is available in Grafana’s Explore interface.
+You can use compose TraceQL queries in Grafana and Grafana Cloud using the Search in Explore and the TraceQL viewer and query editor in the Tempo data source.
```

Comment:
This sentence sounds to me like there is a "Tempo Datasource" screen that I can navigate to and write my TraceQL queries but I don't think that exists. It also suggest there is a view called "query editor", is it a reference to the trace query visual editor of the "Query Type" tab "Search"?

My understanding is that I have to
* Navigate to Explore
* Select a Tempo data source
* Navigate to the "TraceQL" tab. An alternative being to use the TraceQL visual builder of the "Search" tab.

Reply:
Thank you for the clarifications. I've reworded the content using your instructions. 

Query editor was the term used when we only had one location (TraceQL tab) to create TraceQL queries. I'll recheck the phrasing in this article ot make sure that the wording is correct. Thank you for the question.